src/components/Textarea/Textarea.js

```diff
@@ -1,14 +1,40 @@
 // components/Task.js
 import * as React from 'react';
-import { SafeAreaView, View, StyleSheet,TextInput } from 'react-native';
+import {SafeAreaView, View, StyleSheet, TextInput} from 'react-native';
 // import { styles } from '../constants/globalStyles';
 
-export default function Example({phrase,editable,onChange = () => null, placeholder}) {
-
+export default function Example({
+  phrase,
+  editable,
+  onChange = () => null,
+  placeholder,
+  backgroundColor,
+  textColor,
+  border,
+}) {
   return (
-    <SafeAreaView style={styles.container}>
+    <SafeAreaView
+      style={[
+        styles.container,
+        {
+          backgroundColor: backgroundColor ? backgroundColor : '#fff',
+          borderColor: border ? border : '#E5E5E5',
+        },
+      ]}>
       <TextInput
-        style={editable ? styles.input : styles.textarea}
+        placeholderTextColor={
+          textColor === '#111827' ? 'rgba(17, 24, 39, 0.5)' : textColor
+        }
+        style={
+          editable
+            ? [
+                styles.input,
+                {
+                  color: textColor ? textColor : '#111827',
+                },
+              ]
+            : [styles.textarea, {color: textColor ? textColor : '#111827'}]
```

Comment:
This is the only way of approaching it. Maybe you do not understand the logic here. I am doing this because, for the components that do not have ` textColor` that passed to it, they use `"#111827"` color as a default. I what you mean about creating a variable but I don't think it's necessary.